applications/GeoMechanicsApplication/tests/test_element_lab.py

```diff
@@ -0,0 +1,102 @@
+        stage_nr = 0
+        effective_stresses_xx = [integration_point[0,0] for element in effective_stresses_stages[stage_nr] for integration_point in element]
+        effective_stresses_yy = [integration_point[1,1] for element in effective_stresses_stages[stage_nr] for integration_point in element]
+        effective_stresses_zz = [integration_point[2,2] for element in effective_stresses_stages[stage_nr] for integration_point in element]
+        # Assert integration point information
+        for idx, effective_stress_xx in enumerate(effective_stresses_xx):
+            self.assertAlmostEqual(-100.0, effective_stress_xx,        3)
+            self.assertAlmostEqual(-100.0, effective_stresses_yy[idx], 3)
+            self.assertAlmostEqual(-100.0, effective_stresses_zz[idx], 3)
+
+        stage_nr = 1
+        effective_stresses_xx = [integration_point[0,0] for element in effective_stresses_stages[stage_nr] for integration_point in element]
+        effective_stresses_yy = [integration_point[1,1] for element in effective_stresses_stages[stage_nr] for integration_point in element]
+        effective_stresses_zz = [integration_point[2,2] for element in effective_stresses_stages[stage_nr] for integration_point in element]
+        # Assert integration point information
+        for idx, effective_stress_xx in enumerate(effective_stresses_xx):
+            self.assertAlmostEqual(-100.0, effective_stress_xx,        2)
+            self.assertAlmostEqual(-300.0, effective_stresses_yy[idx], 2)
+            self.assertAlmostEqual(-100.0, effective_stresses_zz[idx], 2)
+
+    def test_oedometer_ULFEM(self):
+        test_name = 'oedometer_ULFEM'
+        project_path = test_helper.get_file_path(os.path.join('test_element_lab', test_name))
+        simulation = test_helper.run_kratos(project_path)
+        effective_stresses = test_helper.get_cauchy_stress_tensor(simulation)
+        displacements = test_helper.get_displacement(simulation)
+
+        effective_stresses_xx = [integration_point[0,0] for element in effective_stresses for integration_point in element]
+        effective_stresses_yy = [integration_point[1,1] for element in effective_stresses for integration_point in element]
+        effective_stresses_zz = [integration_point[2,2] for element in effective_stresses for integration_point in element]
+
+        # Assert integration point information
+        for idx, effective_stress_xx in enumerate(effective_stresses_xx):
+            self.assertAlmostEqual(0.0,      effective_stress_xx,        3)
+            self.assertAlmostEqual(-1000000, effective_stresses_yy[idx], 2)
+            self.assertAlmostEqual(0.0,      effective_stresses_zz[idx], 3)
+
+        y_displacements = [displacement[1] for displacement in displacements]
+        top_node_nbrs = [1]
+        for top_node_nbr in top_node_nbrs:
+            self.assertAlmostEqual(-0.0909090909516868, y_displacements[top_node_nbr], 6)
+
+    def test_oedometer_ULFEM_diff_order(self):
```

Comment:
Can you describe what the test does in the docstring?

Reply:
I added a description for the test